### PR TITLE
B-03515 regularly log disk usage

### PR DESCRIPTION
**Pull request by robbiecarlton**
This ended up being reduced in scope, because the way holochain stores data is about to change to sqlite. So now it just gets a mock value for the disk usage per dna. But it does actually store it in service logger

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -179,17 +181,8 @@ class Envoy {
     this.ws_server.on("connection", async (socket, request) => {
       // path should contain the HHA ID and Agent ID so we can do some checks and alert the
       // client-side if something is not right.
-      log.silly("Incoming connection from %s", request.url);
       const url = new URL(request.url, "http://localhost");
 
-      socket.on("message", (data) => {
```
Comment (JettTech) [inline src/index.ts]: why are we removing this?

File: src/index.ts [hunk 2]
```diff
@@ -1006,12 +1039,36 @@ class Envoy {
...
+    log.info("Retrieve Servicelogger cell id using the Installed App Id: '%s'", servicelogger_installed_app_id);
+    const appInfo = await this.callConductor("app", { installed_app_id: servicelogger_installed_app_id });
+
+    if (!appInfo) {
+      log.error("Failed during Servicelogger AppInfo lookup: %s", appInfo);
+      throw new HoloError("Failed to fetch AppInfo for Servicelogger")
+    }
+
+    log.debug("Servicelogger app_info: '%s'", appInfo);
+    // We are assuming that servicelogger is a happ and the first cell is the servicelogger DAN
```
Comment (JettTech) [inline src/index.ts]:
```suggestion
    // We are assuming that servicelogger is a happ and the first cell is the servicelogger DNA
```
I realize I had left a typo in this. :laughing:

File: src/index.ts [hunk 3]
```diff
@@ -1006,12 +1039,36 @@ class Envoy {
     return resp;
   }
 
+  async getServiceLoggerCellId(hha_hash) {
```
Comment (JettTech) [inline src/index.ts:1079]: I like the extraction of this for general reuse.  - Are we using this in more than one place though?  I saw your comment that it breaks in `logServiceConfirmation`, but do you have any leads on why?

**Review by JettTech — APPROVED** [review]
Overall looks good, just have a couple non blocking Qs. :+1:

File: src/index.ts (hunk 1 above)
Comment (robbiecarlton) [reply src/index.ts]: I thought I'd added that for my own logging, so I took it out. I'll put it back.

File: src/index.ts (hunk 3 above)
Comment (robbiecarlton) [reply src/index.ts:1079]: No. I did some investigation but couldn't figure it out. It didn't make sense to me. So currently we're just using it in one place. I think it's still worth extracting it though as it makes sense as a small self-contained function.

**Pull request merged**